pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "pydp"
+version = "0.1.0"
+description = ""
+authors = ["Chinmay Shah <chinmayshah3899@gmail.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.5"
+jupyter = "*"
+pandas = "*"
+
+[tool.poetry.dev-dependencies]
+setuptools = "*"
+bumpversion = "==0.5.3"
+pytest = "*"
+black = {version = "==19.10b0",python = "^3.6"}
+twine = {version = "*", python = "^3.6"}
```

Comment:
twine is required to upload the package, so we need it for py35 also

Reply:
sounds good, I'll remove that python version specification for it